### ISSUE TITLE
Error channel.id

### ISSUE THREAD

**Issue by JustRoys** [RoqueDEV/fivemserverstatus#39]
Hoi,

Ik heb het volgende probleem als ik de bot wil opstarten. Heb de bot heel lang gebruikt maar krijg dit opeens.
![errorstatus](https://user-images.githubusercontent.com/59421443/116146559-e7fcff80-a6de-11eb-8372-b6fc62b75700.PNG)

**Comment by Senlar** [comment]
Same as my Ticket #38

**Comment by JustRoys** [comment]
> Same as my Ticket #38

You found a solution?

Add me on discord
JustRoy®#3301

**Comment by Senlar** [comment]
No, I found the problem but I can't fix it because I'm not a bot dev.

**Comment by Duckerr** [comment]
@JustRoys Fixed with my PR #40

**Comment by sagnikoni** [comment]
Update
